foundation-models/openai/src/test/java/com/sap/ai/sdk/foundationmodels/openai/OpenAiClientTest.java

```diff
@@ -277,16 +300,23 @@ void chatCompletion(@Nonnull final Callable<OpenAiChatCompletionOutput> request)
                   equalToJson(
                       """
                       {
-                        "messages" : [ {
-                          "role" : "system",
-                          "content" : "You are a helpful AI"
-                        }, {
-                          "role" : "user",
-                          "content" : [ {
-                            "type" : "text",
-                            "text" : "Hello World! Why is this phrase so famous?"
-                          } ]
-                        } ]
+                             "temperature" : 1,
+                             "top_p" : 1,
+                             "stream" : false,
+                             "presence_penalty" : 0,
+                             "frequency_penalty" : 0,
+                             "messages" : [ {
+                               "content" : "You are a helpful AI",
+                               "role" : "system"
+                             }, {
+                               "content" : "Hello World! Why is this phrase so famous?",
+                               "role" : "user"
+                             } ],
+                             "logprobs" : false,
+                             "n" : 1,
+                             "parallel_tool_calls" : true,
+                             "tools" : [ ],
+                             "functions" : [ ]
```

Comment:
(Comment)

Diff in payload being sent.
May require further finetuning to stop default values and/or empty arrays.